client/src/components/Header/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
```

Comment:
Why this?

Reply:
How will you bring history prop in this component, can't really work with 'Link' on onClick...